includes/api/class-wc-rest-products-controller.php

```diff
@@ -1180,8 +1178,8 @@ protected function save_product_meta( $product, $request ) {
 			$product->set_parent_id( $request['parent_id'] );
 		}
 
-		// Update parent if grouped so price sorting works and stays in sync with the cheapest child.
-		if ( $product->get_parent_id() > 0 || $product->is_type( 'grouped' ) ) {
+		// Update parent if grouped so price sorting works and stays in sync with the cheapest child. @todo
```

Comment:
Isn't this todo complete by the sync code in grouped products? If so - we should remove the whole block